Update exportPathMap.md (#18483)
vercel/next.js: docs/api-reference/next.config.js/exportPathMap.md

# exportPathMap

> > This feature is meant to be used in conjunction with `next export`. Please refer to [Static HTML export](/docs/advanced-features/static-html-export.md) if you want to learn more about it.

Reviewer: This change is not required, the feature is indeed exclusive of `next export`, it also works with `next dev` but it will only work when exporting the app, and the way you build an app when using `next export` is a bit different from `next build && next start`

```suggestion
> This feature is exclusive of `next export`. Please refer to [Static HTML export](/docs/advanced-features/static-html-export.md) if you want to learn more about it.
```

  </ul>
</details>

> `exportPathMap` allows you to specify a mapping of request paths to page destinations, to be used during export. Paths defined in `exportPathMap` will also be available when running `yarn develop`.

Reviewer: ```suggestion
`exportPathMap` allows you to specify a mapping of request paths to page destinations, to be used during export. Paths defined in `exportPathMap` will also be available using [`next dev`](/docs/api-reference/cli.md#development).
```
